Domain.Sql.Tests/AzureDatabaseConfigurationTests.cs

```diff
@@ -94,19 +94,50 @@ public void AzureSqlDatabase_ReadModel_can_be_configured_using_a_migration()
                 new ReadModelDatabaseInitializer<MigrationsTestReadModels>().InitializeDatabase(context);
 
                 var migrator = new AzureSqlDbMigrator(
-                    serviceObjective: "S0",
-                    edition: "standard",
-                    maxSize: "500 MB",
+                    serviceObjective: "P1",
```

Comment:
Because going from S0 to P1, is where Azure behind the scene need to make a copy of the DB into a new storage. We can delete the DB after the test to avoid $$.